build.gradle

```diff
@@ -429,12 +358,24 @@ integTest {
         is_https = is_https == null ? "true" : is_https
         user = user == null ? "admin" : user
         password = password == null ? "admin" : password
```

Comment:
According to [this issue](https://github.com/opensearch-project/security/issues/3624), don't we need to change the default password from `admin`?

Reply:
no since we're not directly using the install demo configuration script

Reply:
Reference : https://github.com/opensearch-project/flow-framework/issues/310#issuecomment-1908844576

Reply:
I see, thanks for the reference. Let's add the comment in the code as this is bit confusing.